src/api.py

```diff
@@ -0,0 +1,24 @@
+import falcon
+import json
+from util.json_operations import toJSON
+from arena import JsonArena
+from combatants import CombatantNotFoundException
+
+
+class CombatantResource:
+
+    def on_post(self, req: falcon.Request, resp: falcon.Response):
+        raw_data = json.load(req.bounded_stream)
+        arena = JsonArena(max_rounds=int(raw_data['max_rounds']))
+        try:
```

Comment:
Please try to have as few as possible lines of code inside the `try` block (preferably: 1)